zuul-core/src/test/java/com/netflix/netty/common/proxyprotocol/ElbProxyProtocolChannelHandlerTest.java

```diff
@@ -163,7 +159,7 @@ public void negotiateProxy_ppv1_ipv4() {
         EmbeddedChannel channel = new EmbeddedChannel();
         channel.pipeline().addLast(ElbProxyProtocolChannelHandler.NAME, new ElbProxyProtocolChannelHandler(registry, true));
         ByteBuf buf = Unpooled.wrappedBuffer(
-                "PROXY TCP4 192.168.0.1 124.123.111.111 10008 443\r\n".getBytes(StandardCharsets.US_ASCII));
+                "PROXY TCP4 192.168.0.1 124.123.111.111 10008 7007\r\n".getBytes(StandardCharsets.US_ASCII));
```

Comment:
This looks messed up.    The destination port is supposed to be the one the client originally connected to, not the final port where it is going.     Thats why the public IP address is matched with the public https port

Reply:
Yeah, you're right. This should be the one that the client connects to, not the listener port. Updated 
The `LOCAL_ADDR` might need renaming, esp. since the meaning changes, when a HAPM is involved.